db/pebble/db.go

```diff
@@ -38,11 +39,16 @@ func NewMem() (db.DB, error) {
 }
 
 // NewMemTest opens a new in-memory database, panics on error
-func NewMemTest() db.DB {
+func NewMemTest(t *testing.T) db.DB {
```

Comment:
This is the key change.

Reply:
I think you should also add `t.Helper()` as well.